airflow-core/src/airflow/api_fastapi/core_api/routes/ui/config.py

```diff
@@ -59,6 +61,11 @@ def get_configs() -> ConfigResponse:
         "external_log_name": getattr(task_log_reader.log_handler, "log_name", None),
     }
 
+    additional_config["plugins_extra_menu_items"] = [
+        AppBuilderMenuItemResponse.model_validate(mi)
+        for mi in (plugins_manager.flask_appbuilder_menu_links or [])
+    ]
```

Comment:
```suggestion
```
Instead of doing this, you can just add additional key into `additional_config` as below:

```
additional_config: dict[str, Any] = {
        "instance_name": conf.get("api", "instance_name", fallback="Airflow"),
        "test_connection": conf.get("core", "test_connection", fallback="Disabled"),
        "dashboard_alert": DASHBOARD_UIALERTS,
        "show_external_log_redirect": task_log_reader.supports_external_link,
        "external_log_name": getattr(task_log_reader.log_handler, "log_name", None),
        "plugins_extra_menu_items": plugins_manager.flask_appbuilder_menu_links or []
    }
```
I tried this and tests passed. Can you also try this?